Makefile

```diff
@@ -50,7 +50,10 @@ deploy: images
 reload-images: build images
 	./scripts/$@ --restart $(restart)
 
-bin/%/submariner-gateway: vendor/modules.txt main.go $(shell find pkg -not \( -path 'pkg/globalnet*' -o -path 'pkg/routeagent*' \))
+pkg/natdiscovery/proto/natdiscovery.pb.go: pkg/natdiscovery/proto/natdiscovery.proto
```

Comment:
For some odd reason I can’t make an actual suggestion, but this would be better as

    %.pb.go: %.proto